mssql-ballerina/types.bal

```diff
@@ -0,0 +1,56 @@
+// import ballerina/jballerina.java;
```

Comment:
Missing license headers and remove all commented lines

Reply:
Fixed with [`19b168a`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/19b168ad9bdf7ab04eee060816cf5209a2a80729)